pkg/controller/ratelimiter/ratelimiter.go

```diff
@@ -44,6 +44,6 @@ func NewRateLimiter() ratelimiter.RateLimiter {
 func RequeueRateLimiter() ratelimiter.RateLimiter {
 	return workqueue.NewMaxOfRateLimiter(
 		// 5 qps, 50 bucket size.  This is the overall factor, and must be slower than the NewRateLimiter limit, to leave "room" for new items.
```

Comment:
nit: Could you also update the comment. "50" will be confusing. 😆

Reply:
Maybe use an [`ItemExponentialFailureRateLimiter`](https://pkg.go.dev/k8s.io/client-go/util/workqueue#ItemExponentialFailureRateLimiter) ?  (Well, I know it makes the mock test harder to control)